packages/clarity-decode/src/data.ts

```diff
@@ -101,6 +101,7 @@ export function envelope(tokens: Data.Token[]): Data.Envelope {
         sessionId: tokens[6] as string,
         pageNum: tokens[7] as number,
         upload: tokens[8] as Data.Upload,
-        end: tokens[9] as Data.BooleanFlag
+        end: tokens[9] as Data.BooleanFlag,
+        dob: tokens[10] as string
```

Comment:
If this is in the cookie why needed here?

Reply:
It's a 1P cookie, not being sent to the server.